packages/portal/src/cachers/collections/organisations.js

```diff
@@ -17,24 +24,59 @@ async function getRecordCounts() {
   return response.data?.facets?.[0]?.fields || [];
 }
 
+async function getCountryPrefLabel(entityUrl) {
+  const response = await axiosClientEntity.get(entityUrl);
+  return response.data.prefLabel;
+}
+
 const data = async(config = {}) => {
   const organisationData = await baseData({ type: 'organization' }, config);
 
   axiosClient = createEuropeanaApiClient(config.europeana?.apis?.record);
+  axiosClientEntity = createEuropeanaApiClient(config.europeana?.apis?.entity);
 
   const recordCounts = await getRecordCounts();
 
+  // Get array with all unique countries to only have to request prefLabels once
+  const organisationCountries = uniq(organisationData.map(organisation => organisation.country));
+
+  const organisationCountriesPrefLabels = {};
+  for (const country of organisationCountries) {
+    // Acceptance Entity API returns entity URI for country
+    if (isEntityUri(country)) {
+      const entityId = country.split('/').pop();
+      organisationCountriesPrefLabels[country] = await getCountryPrefLabel(`/place/${entityId}.json`);
+    } else if (country) {
+      // Production Entity API returns country code. This as in between solution. No support for Maltese (mt)
```

Comment:
No support for Maltese? It's in the list on the pkg readme: https://www.npmjs.com/package/i18n-iso-countries

Reply:
Excellent!